app/views/incomeSources/cease/CeaseIncomeSources.scala.html

```diff
@@ -64,7 +59,9 @@
     else controllers.incomeSources.cease.routes.DeclarePropertyCeasedController.show(ForeignProperty).url
 }
 
-@noneToUnknown(value:Option[String]) = @{value.getOrElse(getMessage("unknown"))}
+@noneToUnknown(value: Option[String]) = @{
+    value.getOrElse(getMessage("unknown"))
```

Comment:
just wondering if we need to do any changes to support Welsh?

Reply:
This gets message "cease-income-sources.unknown" which is in both messages files